test/spec.yml

```diff
@@ -124,6 +125,13 @@
             "bn invoke momentousgiants"
             "curl -H X-Binaris-Api-Key:* https://*"
 
+- test: Test show
+  steps:
+    - in: BINARIS_ACCOUNT_ID=1066 bn show --account_id
```

Comment:
I would strongly prefer that a file based test also exists.

Reply:
Not done.  I'm not testing that `getAccountId` works here -- if it doesn't, _nothing_ will work.